testing/e2e/cluster_test.go

```diff
@@ -90,18 +86,25 @@ func TestMain(m *testing.M) {
 
 // TestConfigMatch covers that the config file used for local in memory e2e
 // tests matches the configs used for the in cluster tests, to avoid drift.
-func TestConfigMatch(t *testing.T) {
-	cfg, err := config.Read()
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	cfgMemory := viper.New()
-	cfgMemory.SetConfigType("yaml")
-	err = cfgMemory.ReadConfig(strings.NewReader(configFile))
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	require.Equal(t, cfgMemory.AllSettings(), cfg.AllSettings())
-}
+// joeholley: commented out 2023.08.15: this test can no longer work as-is; the standalone
```

Comment:
If this test doesn't work, then let's just delete it.

Reply:
SGTM